public/apps/configuration/panels/get-started.tsx

```diff
@@ -214,6 +214,29 @@ export function GetStarted(props: AppDependencies) {
             </EuiButton>
           </EuiText>
         </EuiPanel>
+
+        <EuiSpacer size="l" />
+
+        <EuiPanel paddingSize="l">
+          <EuiTitle size="s">
+            <h3>Optional: Purge cache</h3>
+          </EuiTitle>
+          <EuiText size="s" color="subdued">
+            <p>
+              By default, the security plugin caches authenticated uses, along with their roles and
+              permissions. This option will purge cached users, roles and permissions.
```

Comment:
Just double check: tenant is not related.

Reply:
Nope only information from external IDP are refreshed.

Reply:
In this case, shall we make it clear in the message? I mean the external IDP part.

Reply:
I don't think we need to go into that detail.